network/server/tchannelthrift/node/service.go

```diff
@@ -207,6 +212,10 @@ func (s *service) FetchBlocksMetadata(tctx thrift.Context, req *rpc.FetchBlocksM
 			blockMetadata := rpc.NewBlockMetadata()
 			blockMetadata.Start = xtime.ToNanoseconds(fetchedMetadataBlock.Start())
 			blockMetadata.Size = fetchedMetadataBlock.Size()
+			if checksum := fetchedMetadataBlock.Checksum(); checksum != nil {
```

Comment:
We're ok with just not having a checksum on the blockMetadata? I'm guessing this is for blocks that are not yet sealed. Is there a better way we can indicate that the block is not sealed, ie with a sealed field instead of empty checksum?